docs/modules/getting_started/partials/proc_connecting-to-host.adoc

```diff
@@ -0,0 +1,34 @@
+= Accessing services from host to {prod}
+
+When you run services on your host, you can configure {prod} to access these services.
+
+.Prerequisites
+* You have a service which is running on your host and want to consume it with {prod}
+* Perform this procedure when using user mode network which is default for Mac/Windows.
+
+
+.Procedure
+. Enable accessing services from host to {prod}:
++
+----
+$ crc config set host-network-access true
+----
+
+. Check {prod} config to make sure correct network-mode and host-network-access are set:
++
+----
+$ crc config view
+[...]
+- network-mode                          : user
+- host-network-access                   : true
+[...]
+----
+
+. If {prod} instance is already running then restart it (stop => start), otherwise just start it.
++
+----
+$ crc stop && crc start
+----
+
+. Assuming your service is running on the host on port `8080`, to access
+it from the {prod} instance, use `host.crc.testing:8080`
```

Comment:
```suggestion
.Verification
. Access the service running on your host from {prod}.
Assuming your service is running on the host on port `8080`, to access it from the {prod} instance, use `host.crc.testing:8080`.
```

Reply:
@themr0c do we really need `Access the service running on your host from {prod}.` part as verification (looks repetitive) ?

Reply:
You are right.